src/btree/bt_walk.c

```diff
@@ -624,3 +625,235 @@ __wti_tree_walk_skip(WT_SESSION_IMPL *session, WT_REF **refp, uint64_t *skipleaf
 
     return (0);
 }
+
+/*
+ * __wt_page_npos --
+ *     Get the page's normalized position in the tree. - \c start is a position within the leaf
+ *     page: - 0 .. 1. If \c path_str_offset is set, make a string representation of the path.
```

Comment:
Add a comment that 0 and 1 are corner cases which **can** return you to an adjacent page.